### ISSUE TITLE
owl.exe does nothing

### ISSUE THREAD

**Issue by VladimirAlexiev** [atextor/owl-cli#15]
I got https://github.com/atextor/owl-cli/releases/download/snapshot/owl-x86_64-windows-snapshot.exe and it doesn't do anything:
```
$ owl -v
$ owl -h
$ owl diagram IEC61970-600-2_CGMES_3_0_0_RDFS_501Ed2CD_EQ.ttl
  # nothing written
```
The same with version 1.2.5.

https://github.com/atextor/owl-cli/releases/download/snapshot/owl-cli-snapshot.jar doesn't have this problem.

**Comment by atextor** [comment]
Hi @VladimirAlexiev, thanks for reporting!
I have moved the issue to owl-cli.
Can you please post the output using hte `-vvv` switch?

**Comment by VladimirAlexiev** [comment]
No output.

(The first time it's run it takes a couple of seconds, then it runs faster)

**Comment by hoijui** [comment]
I am on linux, trying to use this project from source (as I want to do changes afterwards), and I don't know how.
I am used to Maven, not Gradle.

I tried `./gradlew run`, and that .. seems to be it.. somehow, but don't know how to supply CLI commands; `./gradlew run -- -h` seems not to be it.

I also tried to figure out if `./gradlew build` might create a self-contained JAR, but that seems also not to be the case...

in short: how to use this?

**Comment by atextor** [comment]
The documentation contains sections both on [downloading and running a pre-built binary](https://atextor.de/owl-cli/main/snapshot/index.html#download-and-install-owl-cli) as well as [building from source](https://atextor.de/owl-cli/main/snapshot/index.html#building-from-source). Please let me know if this needs improvement though! (apart from the Windows version apparently not doing what it's supposed to)

**Comment by janvlug** [comment]
I confirm this issue Windows 11 Pro with owl-cli-1.2.5. I use the executable. Is it required that java is installed?

**Comment by atextor** [comment]
No. Using the .exe does not require to have Java installed, but the diagram generation requires Graphviz to be installed (i.e., have `dot` in your path).

**Comment by janvlug** [comment]
I installed Windows 11 in a fresh VM (on Linux virtmanager). Next I downloaded owl-x86_64_windows-1.2.5.exe.

When I tried to run owl-x86_64_windows-1.2.5.exe, I got an error on missing vcruntime140.exe.

Next I installed: vcredist_x64.exe.

owl-x86_64_windows-1.2.5.exe did not give the vcruntime140.exe missing error any more when started in a new Command Prompt.

I also installed windows_10_cmake_Release_graphviz-install-12.2.1-win64.exe.

`dot -- help` gives the help text.

`owl-x86_64_windows-1.2.5.exe` still only prints an empty line.

`owl-x86_64_windows-1.2.5.exe -v` also only prints an empty line.

**Comment by janvlug** [comment]
Next I installed Chocolatey.

I ran `choco install graphviz` in an admin PowerShell.

Tried `owl-x86_64_windows-1.2.5.exe -v` again in a fresh admin PowerShell.

Still no output (btw I noticed that the PowerShell does not print empty lines after commands).

**Comment by janvlug** [comment]
Slightly off-topic:

The jar seems to start fine on Windows 11.

First I installed `openjdk` in an admin Powershell with:

`choco install openjdk`

Next in a regular Command Prompt I did in the directory where I downloaded `owl-cli-1.2.5.jar`:

`java -jar owl-cli-1.2.5.jar -h`

and the help text is printed, and the `write` command works as well.

**Comment by atextor** [comment]
Thanks for the detailed report @janvlug!

**Comment by atextor** [comment]
After the larger refactoring of merging owl-cli and turtle-formatter (#20), the project for now does not provide a native binary. It's important to get the project out of the stale status and fix long-standing bugs; after that we can start working again on providing platform-binaries.
In the meantime, please use the executable .jar.
